Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -106,13 +126,12 @@ public async Task<bool> Register()
         /// To use Performance mode. Call UseFindCloudletPerformanceMode(true)
         /// </summary>
         /// <returns>FindCloudletReply Task</returns>
-        public async Task<bool> FindCloudlet()
+        public async Task<bool> FindCloudlet(string dmeHost = null, uint dmePort = 0)
         {
             latestFindCloudletReply = null;
-
+            Debug.Log("FindCloudlet to DME: " + dmeHost);
```

Comment:
you print dmeHost later in the function. don't think you need this

Reply:
Removed nit.